products/angularjs.md

```diff
@@ -2,16 +2,18 @@
 title: AngularJS
 category: framework
 tags: google javascript-runtime
-iconSlug: angular
```

Comment:
The current icon is technically incorrect.
Angular and AngularJS are two different products (yes, the name choosing doesn't help; the community has struggled with that since the release of Angular).

The right one is `angularjs` but since that icon [is going to be removed](https://github.com/simple-icons/simple-icons/pull/9304) from Simple Icons, the End of Life [contributing guidelines](https://github.com/endoflife-date/endoflife.date/blob/master/CONTRIBUTING.md) indicates that there should no `iconSlug`

`CONTRIBUTING.md`
```MD
...
# Remove this property if the icon is not available on Simple Icons.
...
```

This change will help to clarify the confusion between Angular and Angularjs

Reply:
I agree with this change. I am OK to merge this change in this PR, but if you want it to be merge faster you can also open a separate PR (given the nature of the change I am pretty sure it will be merged in less than 1-2 days).